chart-infra/templates/kube-cleanup-operator-deployment.yaml

```diff
@@ -72,6 +72,14 @@ rules:
   - list
   - watch
   - delete
+- verbs:
+  - get
+  - patch
+  apiGroups:
+  - apps
+  resources:
+  - deployments
+  - deployments/scale
```

Comment:
The cleanup operator's service account role is extended to allow for get/patch of deployments for scaling them down to zero.